features/managing_invoices/seeing_invoice_with_items_having_promotion_applied.feature

```diff
@@ -0,0 +1,30 @@
+@managing_invoices
+Feature: Seeing an invoice with items and shipment having promotion applied
+    In order to have items with proper amounts
```

Comment:
I don't want to be picky 😄 But I feel we should work over these features descriptions a little bit 🚀 Having proper amounts is not a goal in the essence, it's more a way to... what? Not having legal problems? 🤯 I can leave with it, but I know we can do better 😄

Reply:
Yup, you're 😄 For me, the intention of Administrator to have proper amounts on the invoice is enough, but feel free to refactor these features 😃